spec/models/manageiq/providers/vmware/infra_manager/vm/remote_console_spec.rb

```diff
@@ -155,15 +155,17 @@
       EvmSpecHelper.create_guid_miq_server_zone
       ems.update(:ipaddress => '192.168.252.14', :hostname => '192.168.252.14')
       auth = FactoryBot.create(:authentication,
-                                :userid   => 'dev1',
-                                :password => 'dev1pass',
-                                :authtype => 'console')
+                               :userid   => 'dev1',
+                               :password => 'dev1pass',
+                               :authtype => 'console')
       ems.authentications = [auth]
       ticket = VCR.use_cassette(described_class.name.underscore) do
         vm.remote_console_vmrc_acquire_ticket
       end
+
       expect(ticket).to have_key(:ticket)
       expect(ticket[:ticket]).to match(/^[0-9\-A-Z]{40}$/)
+      expect(ticket[:ticket].class.name).to eq("String") # Ensure that VimStrings aren't returned
```

Comment:
Slightly clearer

```suggestion
      expect(ticket[:ticket]).to be_instance_of("String") # Ensure that VimStrings aren't returned
```

Reply:
Ah didn't do that because I thought VimString (subclass of String) might return true but looks like that isn't the case